### PR TITLE
#21291 Fix ecr vulnerabilities thecodingmachine/gotenberg

### PR DESCRIPTION
**Pull request by andika-william**
## 🌊 Changes

- Update thecodingmachine/gotenberg to current latest; [version 8.2.2](https://github.com/gotenberg/gotenberg/releases/tag/v8.2.2)
- Updated as part of the fix ECR issue in https://github.com/vulcan-ai/vulcan/pull/21788

### CODE REVIEW

File: charts/datasaur/templates/start/document-converter-deployment.yaml [hunk 1]
```diff
@@ -32,9 +32,9 @@ spec:
 {{ include "imagePullSecretTemplate" . | nindent 6 }}
       containers:
 {{- if .Values.dockerHub.enabled }}
-        - image: thecodingmachine/gotenberg:7.10.2
+        - image: thecodingmachine/gotenberg:8.2.2
 {{- else }}
-        - image: {{ .Values.image.registry | trimSuffix "/" }}{{ if .Values.image.registry }}/{{ end }}thecodingmachine/gotenberg:7.10.2
+        - image: {{ .Values.image.registry | trimSuffix "/" }}{{ if .Values.image.registry }}/{{ end }}thecodingmachine/gotenberg:8.2.2
```
Comment (tonoman3g) [inline charts/datasaur/templates/start/document-converter-deployment.yaml:37]: tolong apply changes dr PR mike juga ya @andika-william 

https://github.com/datasaur-ai/helm-charts/pull/29/files

Comment (tonoman3g) [reply charts/datasaur/templates/start/document-converter-deployment.yaml:37]: keep up master dulu yak @andika-william

Comment (andika-william) [reply charts/datasaur/templates/start/document-converter-deployment.yaml:37]: okede, on my way @tonoman3g

Comment (andika-william) [reply charts/datasaur/templates/start/document-converter-deployment.yaml:37]: rebased to current master (main)

aku liat PR audrey yg merged juga sudah keeup sama punya michael. 
Jadi harusnya master/main sudah keeup juga ya ka ton 👀

**Review by daniel-jeremia — COMMENTED** [review]
I made a hotfix to main because of a breaking bug.. please keep up to main branch 🙏 @andika-william

**Comment by daniel-jeremia** [comment]
@tonoman3g if we merge this, then the image for gotenberg will only be provided from datasaur version >=6.56.0 (because the vulcan side is merged to master).. For now it shouldn't affect any customers sih since we enable dockerHub by default, but if we have air-gapped customers (no access to Docker Hub) they will be affected.

**Comment by tonoman3g** [comment]
> @tonoman3g if we merge this, then the image for gotenberg will only be provided from datasaur version >=6.56.0 (because the vulcan side is merged to master).. For now it shouldn't affect any customers sih since we enable dockerHub by default, but if we have air-gapped customers (no access to Docker Hub) they will be affected.

agreed with your concern, we will tackle this in the Q2 roadmap item. 

so for now, @andika-william please add minimum version in the helm chart to >= 6.56.0

**Comment by andika-william** [comment]
okede adjusted here ya guys @tonoman3g @daniel-jeremia 

https://github.com/datasaur-ai/helm-charts/pull/30/commits/83c8013dbb16ab4ac71d8676dfec7e48524700ec: update chart.yaml to appVersion >=6.56.0
https://github.com/datasaur-ai/helm-charts/pull/30/commits/205843f2ae0efc595588a2c1864aa0c2893af5ff: regenerate index.yaml and package

**Review by tonoman3g — APPROVED** [review]

**Pull request merged**